model/metadata.hpp

```diff
@@ -69,6 +69,9 @@ class metadata {
     const std::string& description(void) const;
     bool has_cleanup(void) const;
     bool is_exclusive(void) const;
+    const std::string& execenv(void) const;
+    bool has_execenv(void) const;
+    const std::string& execenv_jail(void) const;
```

Comment:
This should be put directly below `description` to be sorted alphanumerically.

Reply:
Fixed.